tests/experimental/metric_repository/test_column_descriptive_metrics_metric_retriever_integration.py

```diff
@@ -16,8 +16,8 @@
     TableMetric,
 )
 
-# Import and use fixtures defined in tests/datasource/fluent/conftest.py
-from tests.datasource.fluent.conftest import (
+# Import and use fixtures defined in tests/conftest.py
+from tests.conftest import (
```

Comment:
Same as above/below

Reply:
Addressed in https://github.com/great-expectations/great_expectations/pull/8773/commits/3e17393f0dc563d9cb3a01b571d5b3f97563eb64